runtests.sh

```diff
@@ -77,7 +77,7 @@ run_integration(){
         export SNAPCRAFT=snapcraft-coverage
     fi
 
-    python3 -m unittest discover -b -v -s integration_tests
+    SNAPCRAFT_FROM_SOURCE=1 python3 -m unittest discover -b -v -s integration_tests
```

Comment:
Will this break the ability to run the tests directly while developing? (e.g. `python3 -m unittest snapcraft.tests.foo`)